data/lua/TLoZ/json.lua

```diff
@@ -0,0 +1,380 @@
+--
+-- json.lua
+--
+-- Copyright (c) 2015 rxi
+--
+-- This library is free software; you can redistribute it and/or modify it
+-- under the terms of the MIT license. See LICENSE for details.
```

Comment:
the MIT License doesn't have many requirements, but the one it does have is for the license file to accompany this file, which it does not in this PR.